docs/content/maintenance/manage-tags.md

```diff
@@ -108,7 +108,7 @@ You can create a tag with given name (cannot be number) and snapshot ID.
     --database <database-name> \ 
     --table <table-name> \
     --tag_name <tag-name> \
-    --snapshot <snapshot-id> \
+    [--snapshot <snapshot_id>] \
```

Comment:
You need to modify Flink and Spark procedure too.

Reply:
done.